x-pack/test/functional/es_archives/security/feature_privileges/mappings.json

```diff
@@ -0,0 +1,438 @@
+{
+  "type": "index",
+  "value": {
+    "index": ".kibana_2",
```

Comment:
Is there a specific reason we're using `.kibana_2` here and elsewhere? I don't see this pattern in existing archiver mappings

Reply:
I can go through and change them... I used the esarchiver to create these, and the migrations "did this for us"